about.html

```diff
@@ -0,0 +1,145 @@
+              <li>
+                <a href="https://www.facebook.com/ArabAffiliateSummitOfficial/" target="_blank" rel="noopener">
+                  <i class="fa fa-facebook-f"></i>
+                </a>
+              </li>
+              <li><a href="https://www.facebook.com/ArabAffiliateSummitOfficial/" target="_blank" rel="noopener">
+                <i class="fa fa-twitter"></i>
+              </li>
+              <li><a href="http://arabaffiliatesummit.com/" target="_blank" rel="noopener">Join</li>
+              <li><a href="http://arabaffiliatesummit.com/" target="_blank" rel="noopener">Program</a></li>
+            </ul>
+          </div>
+        <nav>
+          <div class="nav-links">
+            <div class="hamburger">
+              <img
+              src="./img/Enabled.png"
+              class="hamburger-button"
+              alt="Mobile menu icon"
+            />
+            </div>
+            <!-- logo -->
+          <div href="/index.html">
+            <a class="logo"href="#">Arab Affiliate Morocco</a>
+          </div>
+          <ul class="menu-list">
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a>Program</a></li>
+            <li><a>Join</a></li>
+            <li><a>Sponsors</a></li>
+            <li><a>BOOK TICKETS</a></li>
+          </ul>
+          </div>
+        </nav>
+        <div class="menu-overlay"></div>
+        <div class="menu-list-mobile">
+          <ul class="menu-list-mobile-2">
+            <li class="close-menu-button">
+              <img src="./img/menu-close.png" class="close-btn" alt="Close menu button" />
+            </li>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsors</a></li>
+          </ul>
+        </div>
+        <div class="about-welcome">
+            <p class="about-welcome-text center">"Hello! Sharing world."</p>
+            <h1 class="about-header center">
+              ARAB AFILIATE SUMMIT MOROCCO <br /> ASSEMBLY 2022
+            </h1>
+            <p class="about-message center">
+             Lorem ipsum dolor sit amet consectetur adipisicing elit. tempore quasi saepe harum nostrum voluptates sint provident fuga ea ex alias natus reiciendis, iste perspiciatis mollitia. Architecto esse exercitationem accusantium, aspernatur eligendi explicabo eveniet praesentium animi minus neque assumenda.
+            </p>
+            <div class="contact">
+              <p class="contact-text center">
+                Please contact us per Email for any further questions about the
+                Arab Affiliate Morocco Nov 2022
+              </p>
+              <p class="contact-email center">arabafiliatesummit@sumit2022.org</p>
+            </div>
+          </div>
+      </header>
```

Comment:
- [ ] It appears that the images including the logo in the content area are overlapping with the desktop navigation bar. Consider fixing this small issue.